code/game/objects/items/research_upgrades.dm

```diff
@@ -0,0 +1,72 @@
+//prop items
+/obj/item/XenoBio
+	name = "An unidentified Alien Organ"
+	desc = "Looking at it makes you want to vomit"
+	icon = 'icons/obj/items/Marine_Research.dmi'
+	icon_state = "biomass"
+	black_market_value = 50
+	//For all of them for now, until we have specific organs/more techs
+
+/obj/item/XenoBio/Resin
+	name = "Alien Resin"
+	desc = "A piece of alien Resin"
+	icon_state = "biomass"
+
+
+/obj/item/XenoBio/Chitin
+	name = "Alien Chitin"
+	desc = "A chunk of alien Chitin"
+	icon_state = "chitin-chunk"
+
+
+/obj/item/XenoBio/Blood
+	name = "Alien Blood"
+	desc = "A sample of alien Blood"
+	icon_state = "blood-vial"
+
+//prop items end
+
+
+
+//previously file holding left over stuff that never got finished from 8 years ago, it was boring though, so we change that.
+//changing to datums
+/obj/item/research_upgrades
+	name = "Research upgrade"
+	desc = "Somehow you got this, you shouldnt be able to, consider yourself special."
+	icon = 'icons/obj/items/disk.dmi'
+	icon_state = "datadisk1" // doesnt HAVE to be a disk!
+	///technology stored on this disk, goes through one to whatever levels of upgrades there are, set it to last upgrade in your line. 0 Excludes it from buying menu.
+	///price of the upgrade, if a single disk could mean many upgrades, use list in sync with upgrades to set its prices
+	var/value
+/obj/item/research_upgrades/autodoc
+	name = "Research Upgrade (AutoDoc)"
+	desc = "Research upgrade for AutoDoc, Technology on this disk is used "
+	value = UPGRADE_TIER_1
+
+
+/obj/item/research_upgrades/autodoc/get_examine_text(mob/user)
+	. = ..()
+	switch(value)
+
+		if(UPGRADE_TIER_1)
+			. += "for stopping internal bleedings."
+		if(UPGRADE_TIER_2)
+			. += "for fixating and mending broken bones."
+		if(UPGRADE_TIER_3)
+			. += "for treating damaged organs."
+		if(UPGRADE_TIER_4)
+			. += "for extracting unknown parasites."
```

Comment:
this proc isn't for appending text to a one-line `desc`

Reply:
yep thats fixed